### PR TITLE
apk: add German translation

### PR DESCRIPTION
**Pull request by lemeleti**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.de/linux/apk.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# apk
+
+> Alpine Linux-Paketverwaltungstool.
+> Weitere Informationen: <https://wiki.alpinelinux.org/wiki/Alpine_Linux_package_management>.
+
+- Aktualisiere die Indexe von allen externen Repositories:
```
Comment (marchersimon) [inline pages.de/linux/apk.md:6]: The plural is actually `Indizes`.
```suggestion
- Aktualisiere die Indizes von allen externen Repositories:
```

File: pages.de/linux/apk.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`apk update`
+
+- Installiere ein neues Paket:
+
+`apk add {{Paket}}`
```
Comment (marchersimon) [inline pages.de/linux/apk.md:12]: Arguments sholud be lowercase everywhere
```suggestion
`apk add {{paket}}`
```

File: pages.de/linux/apk.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Entferne ein Paket:
+
+`apk del {{Paket}}`
+
+- Reparieren Sie das Paket oder aktualisieren es, ohne die Hauptabhängigkeiten zu ändern:
```
Comment (marchersimon) [inline pages.de/linux/apk.md:18]:
```suggestion
- Repariere oder aktualisiere ein Paket, ohne die Hauptabhängigkeiten zu ändern:
```

File: pages.de/linux/apk.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`apk fix {{Paket}}`
+
+- Suche Pakete mit einem Schlüsselwort:
+
+`apk search {{Schlüsselwort}}`
+
+- Bekomme Informationen von einem spezifischen Paket:
```
Comment (marchersimon) [inline pages.de/linux/apk.md:26]:
```suggestion
- Erhalte Informationen über eine bestimmtes Paket:
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**